test/e2e/data/infrastructure-azure/v1beta1/bases/calico.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: addons.cluster.x-k8s.io/v1alpha1
+kind: HelmChartProxy
```

Comment:
this seems like a duplicate of the one under templates/addons, thoughts on reusing that one in this folder instead of maintaining two separate files?

Reply:
I was thinking about how to handle that -- using the other one in `templates/` would require going to a different root folder all together, and it seems like most of `test/e2e` and `templates/` are both self contained. Do you think it'd be fine to have a filepath link like that?

Reply:
Fair, looks like we're already deplicating other specs like cluster identity. We can keep it separate for now and look into merging those two test template directories at some point